Challenge1/package.json

```diff
@@ -1,14 +1,14 @@
 {
-  "name": "forta-agent-starter",
-  "displayName": "Forta Agent Starter",
+  "name": "forta-bot-check-bot-deployments",
+  "displayName": "Nethermind Deployment of Forta Bots",
   "version": "0.0.1",
-  "description": "Forta Agent Typescript starter project",
-  "longDescription": "Welcome to the Forta Agent JavaScript starter project!\n\nForta Network presents this starter project as an entry point for bot developers who want to integrate with the Forta Network ecosystem. As part of our commitment to empowering developers, this project provides a foundation for building powerful and secure bots that can interact with the Forta platform and take advantage of its cutting-edge capabilities.\n\nThe Forta Agent SDK and CLI tool offer a seamless experience for developers who wish to leverage the decentralized and trustless environment provided by the Forta Network. Whether you're interested in creating DeFi bots, monitoring smart contract transactions, or generating real-time alerts, this starter project sets you on the right path to achieve your goals.\n\nJoin the Forta Network community and learn more about our mission to revolutionize decentralized finance and blockchain analytics: https://forta.org/",
-  "repository": "https://github.com/forta-network/forta-bot-sdk/tree/master/starter-project",
+  "description": "This bot detects Forta bots deployed by Nethermind.",
+  "repository": "https://github.com/deca12x/FortaBotChallenges/tree/main/Challenge1",
   "licenseUrl": "https://github.com/forta-network/forta-bot-sdk/blob/master/starter-project/LICENSE.md",
   "promoUrl": "https://forta.org",
   "chainIds": [
-    1
+    1,
+    137
```

Comment:
Since the Forta contracts are all on Polygon, this bot would really only need to scan Polygon and not Ethereum mainnet.